src/Simulation/Simulators/ToffoliSimulator/ToffoliSimulator.cs

```diff
@@ -1,15 +1,14 @@
-﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+﻿// Copyright (c) Microsoft Corporation.
```

Comment:
What is the benefit of this change? What are the benefits of this implementation over the existing one? It seems to me that this implementation will be slightly slower and is less extensible.

Reply:
The simulator uses the `IQuantumProcessor` interface. It requires fewer code and is therefore easier to maintain. Further, we expect that the change in data structures, e.g., how to store simulation values, will increase the performance. Also `QuantumProcessorDispatcher` can be extended in the same way as `SimulatorBase`, so we won't loose in extensibility.

Reply:
I compared the proposed implementation to the current implementation using our ToffoliChain performance benchmarks and found no performance degradation, but also no noticeable speed-up. However, we still think to switch to the proposed implementation for more simplicity and easier maintainability.